src/edu/csus/ecs/pc2/clics/API202003/EventFeedStreamer.java

```diff
@@ -418,42 +472,47 @@ public void runChanged(RunEvent event) {
             }
         }
 
+        @Override
         public void runRemoved(RunEvent event) {
 
             Run run = event.getRun();
             Account account = contest.getAccount(run.getSubmitter());
             if (account.isAllowed(Permission.Type.DISPLAY_ON_SCOREBOARD)) {
 
-                String json = getJSONEvent(JUDGEMENT_KEY, getNextEventId(), EventFeedOperation.DELETE, "{id: \"" + jsonTool.getSubmissionId(run) + "}");
+                String json = getJSONEvent(JUDGEMENT_KEY, getNextEventId(), EventFeedOperation.DELETE, "{id: \"" + IJSONTool.getSubmissionId(run) + "}");
```

Comment:
Same comment as above; I'm not sure about invoking static methods in interfaces...

Reply:
Same reply as above.